feat: Align the repo with new name identus-cloud-agent

mineme0110 opened the pull request:
### Description: 
Align the repo with new name identus-cloud-agent.
This is a major breaking change a lot of variable names are changed and all the packets are changed.

### Alternatives Considered (optional): 


### Checklist: 
- [ ] My PR follows the [contribution guidelines](https://github.com/hyperledger/identus-cloud-agent/blob/main/CONTRIBUTING.md) of this project
- [ ] My PR is free of third-party dependencies that don't comply with the [Allowlist](https://toc.hyperledger.org/governing-documents/allowed-third-party-license-policy.html#approved-licenses-for-allowlist)
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked the PR title to follow the [conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0/)

FabioPinheiro commented:
We should wait for https://github.com/hyperledger/identus-cloud-agent/pull/966 to rename the modules:
- castor/lib/core/src
- prism-agent/service/server
- prism-agent/service/wallet-api
- shared/crypto/

We should wait for https://github.com/hyperledger/identus-cloud-agent/pull/968 to rename the modules:
- prism-agent/service/server

yshyn-iohk commented on README.md:
```diff
@@ -3,21 +3,21 @@
   </a>
   <br>
-  <i> <font size="18"> Open Enterprise Cloud Agent </font> </i>
+  <i> <font size="18"> Open Cloud Agent </font> </i>
```
```suggestion
  <i> <font size="18"> Identus Cloud Agent </font> </i>
```

yshyn-iohk commented on README.md:
````diff
@@ -118,57 +118,57 @@ System requirements can vary depending on the use case. The following are the mi
 Here is a general example of running a Cloud Agent locally:
 ```bash
-PORT=${PORT} PRISM_AGENT_VERSION=${PRISM_AGENT_VERSION} PRISM_NODE_VERSION=${PRISM_NODE_VERSION} \
+PORT=${PORT} AGENT_VERSION=${AGENT_VERSION} NODE_VERSION=${NODE_VERSION} \
````
hm...
I guess we should consider another suffix for the environment variables and belong to the agent:
IDENTUS_CLOUD_AGENT_VERSION - is too long
ICA_VERSION - is good from my point of view, ICA abbreviation should be introduced.

mineme0110 replied:
@yshyn-iohk  we are  following the spreadsheet if you have comments  lets discuss here https://docs.google.com/spreadsheets/d/1lR3abdg6qExXasUVqBlFouNu9FCB0I1ynMMZodQ6y8U/edit#gid=0

yshyn-iohk replied:
@mineme0110, let's discuss the renaming there

yshyn-iohk commented on SECURITY.md:
```diff
@@ -2,7 +2,7 @@
 # Reporting Security Issues
 
-The Open Enterprise Agent (OEA) team and community take security bugs in the components of the OEA ecosystem seriously. We appreciate your efforts to responsibly disclose your findings, and will make every effort to acknowledge your contributions.
+The Open Cloud Agent (OEA) team and community take security bugs in the components of the OEA ecosystem seriously. We appreciate your efforts to responsibly disclose your findings, and will make every effort to acknowledge your contributions.
```
```suggestion
The Identus Cloud Agent (ICA) team and community take security bugs in the components of the OEA ecosystem seriously. We appreciate your efforts to disclose your findings responsibly and will make every effort to acknowledge your contributions.
```

yshyn-iohk commented on package.json:
```diff
@@ -123,7 +123,7 @@
           "markdownReleaseNotes": true,
           "onSuccessTemplate": {
-            "text": "A new version of Opent Enterprise Agent successfully released!\nVersion: `$npm_package_version`\nTag: $repo_url/releases/tag/$npm_package_version\n\nRelease notes:\n$release_notes"
+            "text": "A new version of Open Cloud Agent successfully released!\nVersion: `$npm_package_version`\nTag: $repo_url/releases/tag/$npm_package_version\n\nRelease notes:\n$release_notes"
```
```suggestion
            "text": "A new version of Identus Cloud Agent successfully released!\nVersion: `$npm_package_version`\nTag: $repo_url/releases/tag/$npm_package_version\n\nRelease notes:\n$release_notes"
```

yshyn-iohk reviewed (approved):
Good start, @mineme0110!
Could you accept my suggestions?

mineme0110 commented:
> We should wait for #966 to rename the modules:
> 
> * `castor/lib/core`
> * `prism-agent/service/server`
> * `prism-agent/service/wallet-api`
> * `shared/crypto`
> 
> We should wait for #968 to rename the modules:
> 
> * `prism-agent/service/server`
> 
> We should wait for #975 to rename the modules:
> 
> * `pollux/lib/core`
> * `pollux/lib/vc-jwt`
> * `prism-agent/service/server`

Labels are linked with the module names so we have to wait here
https://github.com/hyperledger/identus-cloud-agent/blob/main/.github/labeler.yml

mineme0110 commented:
> Good start, @mineme0110! Could you accept my suggestions?

@yshyn-iohk I have addressed some of your comments but  other comments are contradicting the document we have 
https://docs.google.com/spreadsheets/d/1lR3abdg6qExXasUVqBlFouNu9FCB0I1ynMMZodQ6y8U/edit?pli=1#gid=0

patlo-iog commented on build.sbt:
```diff
@@ -844,32 +837,33 @@ lazy val prismAgentWalletAPI = project
+  .in(file("cloud-agent/service/server"))
   .settings(commonSetttings)
   .settings(
+    name := "cloud-agent",
     name := "identus-cloud-agent",
```
name is defined twice
